src/library/impl/meta_reader/cache.h

```diff
@@ -135,8 +135,6 @@ namespace xlang::meta::reader
 
         void remove_legacy_cppwinrt_foundation_types()
```

Comment:
maybe rename function per it's now narrower purpose - something like remove_projected_numeric_types

Reply:
👍